packages/DataGrid/src/hooks/useGridEventHandler.js

```diff
@@ -38,30 +41,37 @@ export default function useGridEventHandler({
     if (!refContainer.current && cell !== null) return;
 
     return refContainer.current.querySelector(
-      `[data-cell='${gridId}.${cell.current.columnIndex}.${cell.current.rowIndex}']`
+      `[data-pka-cell-key='${gridId}.${cell.current.columnIndex}.${cell.current.rowIndex}']`
     );
   }, [gridId, refContainer]);
 
   const setRefPrevCell = React.useCallback(() => {
     refPrevCell.current = $getCell();
   }, [$getCell]);
 
-  const setHighlight = React.useCallback(() => {
-    if (cell && cell.current && refGrid && refGrid.current) {
-      const $cell = $getCell();
+  const setHighlight = React.useCallback(
+    ({ rowIndex, columnIndex }) => {
+      if (cell && cell.current && refGrid && refGrid.current) {
+        const $cell = $getCell();
+
+        if ($cell) {
+          const classNameStr = "grid--is-active";
```

Comment:
`data-grid__cell--is-active` would probably be a better BEM name, but since Paprika has managed to be very `className` agnostic, maybe we could use a `data-pka` attribute instead?